### PR TITLE
chore: Add connections metadata

### PR DESCRIPTION
**Pull request by q2w**
This PR adds connections metadata for postgresql module

### CODE REVIEW

File: modules/postgresql/metadata.yaml [hunk 1]
```diff
@@ -212,6 +212,18 @@ spec:
               email = string
             }))
         defaultValue: []
+        connections:
+        - source:
+            source: github.com/terraform-google-modules/terraform-google-service-accounts//modules/simple-sa
+            version: v4.2.3
```
Comment (bharathkkb) [inline modules/postgresql/metadata.yaml:218]:
```suggestion
            version: v4.3.0
```

File: modules/postgresql/metadata.yaml [hunk 2]
```diff
@@ -212,6 +212,18 @@ spec:
...
+          spec:
+            outputExpr: id
+        connections:
+        - source:
+            source: github.com/GoogleCloudPlatform/terraform-google-cloud-run//modules/v2
+            version: v0.12.0
```
Comment (bharathkkb) [inline modules/postgresql/metadata.yaml:224]:
```suggestion
            version: v0.13.0
```

**Review by bharathkkb — APPROVED** [review]
Minor suggestions for updating to releases with the expected outputs

File: modules/postgresql/metadata.yaml (hunk 1 above)
Comment (q2w) [reply modules/postgresql/metadata.yaml]: Done!

**Pull request merged**